pom.xml

```diff
@@ -175,64 +171,34 @@
                 <version>1.7.22</version>
             </dependency>
             <dependency>
-                <groupId>org.apache.maven</groupId>
-                <artifactId>maven-embedder</artifactId>
-                <version>3.3.1</version>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-simple</artifactId>
+                <version>1.7.22</version>
             </dependency>
         </dependencies>
     </dependencyManagement>
 
     <build>
-        <extensions>
-            <extension>
-                <groupId>org.jvnet.wagon-svn</groupId>
-                <artifactId>wagon-svn</artifactId>
-                <version>1.8</version>
-            </extension>
-        </extensions>
         <pluginManagement>
             <plugins>
                 <plugin>
+                    <groupId>org.apache.maven</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
+                    <version>3.1</version>
                     <configuration>
                         <source>1.5</source>
                         <target>1.5</target>
                     </configuration>
                 </plugin>
                 <plugin>
+                    <groupId>org.apache.maven</groupId>
                     <artifactId>maven-invoker-plugin</artifactId>
-                    <version>1.5</version>
+                    <version>2.0.0</version>
                 </plugin>
             </plugins>
         </pluginManagement>
         <plugins>
             <plugin>
-                <artifactId>maven-enforcer-plugin</artifactId>
-                <version>1.0</version>
-                <executions>
-                    <execution>
-                        <id>enforce-dependencies</id>
-                        <goals>
-                            <goal>enforce</goal>
-                        </goals>
-                        <configuration>
-                            <rules>
-                                <bannedDependencies>
-                                    <excludes>
-                                        <exclude>junit:junit</exclude>
-                                    </excludes>
-                                    <message>Use junit:junit-dep instead of junit.</message>
-                                </bannedDependencies>
-                                <requireMavenVersion>
-                                    <version>[3.0,4.0)</version>
-                                    <message>Version 2.x of Mojo Executor requires Maven ${maven.version}. Use Mojo Executor 1.x with Maven 2.</message>
-                                </requireMavenVersion>
```

Comment:
Good point ; I was removing the junit check and got a bit over-eager ;-) Frankly, given that Maven 2.x went EOL in 2009 I'd be surprised if people were still using it ;-) I can restore the check I suppose.